R/print.parameters_model.R

```diff
@@ -43,10 +43,21 @@
 #'   print(mp, select = c("Parameter", "Coefficient", "SE"))
 #'
 #'   print(mp, select = "minimal")
-#' }}
+#' }
```

Comment:
Putting these curly braces into multiple lines adds empty rows into the examples from the PDF, thus the original formatting here is intentional.